### PR TITLE
Fix git actions

### PR DESCRIPTION
**Pull request by oodigie**
**Changes:**
- build: added step to install docker-compose v1 in Git actions

### CODE REVIEW

**Review by cjwmorgan-sol — APPROVED** [review]
Git actions are running again and are passing this is good to merge.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -42,7 +42,6 @@ jobs:
       - name: Install Compose
         uses: ndeloof/install-compose-action@v0.0.1
         with:
-          version: v2.1.0 # defaults to 'latest'
```
Comment (cjwmorgan-sol) [inline .github/workflows/test.yml]: We shoudl have a pinned version. And not default to latest. To avoid un reproducible issues.

**Review by cjwmorgan-sol — CHANGES_REQUESTED** [review]
The latest change has removed the pinned docker compose version. Without a pinned version the test can break on update they are not prepared for. we should slect a version and pin it.

File: .github/workflows/test.yml (hunk 1 above)
Comment (oodigie) [reply .github/workflows/test.yml]: I have made the update now.

**Review by cjwmorgan-sol — APPROVED** [review]
Changes look good and git actions are passing again.
This is ready for merge.

**Pull request merged**